### DISCUSSION TITLE
Buttons with symbols

### DISCUSSION THREAD

**Discussion by bjr005** [kachurovskiy/nanoels#115] [Q&A]
I would really like buttons with symbols, like the yellow ones on picture in the H2 readme. Any chance of getting these designs published?
//Björn

**Comment by kachurovskiy** [comment]
Hi Björn, sure - https://raw.githubusercontent.com/kachurovskiy/nanoels/main/h2/h2-panel.pdf - thanks!

**Reply by bjr005** [reply]
Thanks, fastest reply ever 😁
I thought that it was 3D-printed buttons with symbols on, but that's wrong I suppose?

**Reply by kachurovskiy** [reply]
It's done as a graphics overlay - https://www.jrpanel.com/product/GraphicOverlay

You can get similar results by printing it out on a regular paper, cutting out a window in the middle, laminating, embossing buttons with a hot metal dome (perhaps 100-150 degrees controlled with a soldering iron?) and gluing it to the front of the device with a double-sided sticky tape. Distance between button tips and the back of the graphics overlay should be very small, 0.1-0.3mm.
